tests/backwards_compatibility_tests/test_vector_normalisation.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+
+from base_test_case import BaseCompatibilityTestCase
+from marqo_test import MarqoTestCase
+import marqo
+
+
+@pytest.mark.marqo_version('2.13.0')
```

Comment:
What does the mark do? Try to indicate the version that the results are created? If we have marqo 2.20.x later and won't need to test on 2.13.0 since we only test 4 versions, will these mark still be useful?

Reply:
This is used to determine whether a certain test should be executed or not. For more details you can see this: https://www.notion.so/Backwards-compatibility-and-rollback-testing-16d64207c72e486fa25001192f20817b?pvs=4#cf252df2d7214fc3843a02fc395dd4c6